Specs/DateTime/English/DateTimeModel.json

```diff
@@ -21430,5 +21430,205 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Created On is in the first week of 2021",
+    "Context": {
+      "ReferenceDateTime": "2018-11-29T12:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "the first week of 2021",
+        "Start": 17,
+        "End": 38,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2021-W01",
+              "type": "daterange",
+              "start": "2021-01-04",
+              "end": "2021-01-11"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Created On is in the second week of 2021",
```

Comment:
Desirable: It would be great if new specs have more variety in sentence structure than just repeating the base sentence and changing the mention form. Same in other languages.

Reply:
modified inputs